website/server/models/user/schema.js

```diff
@@ -357,6 +358,7 @@ let schema = new Schema({
     party: {type: Schema.Types.Mixed, default: () => {
       return {};
     }},
+    parties: {type: [{}], default: () => []},
```

Comment:
What was the reason for specific values for the properties?  I was thinking of leaving the schema description as a generic array of objects so future developers could make changes to the User model easily.  If your comment was regarding syntax for the type description I pulled it from the docs [(see under 'array')](http://mongoosejs.com/docs/schematypes.html)

Assuming the syntax is correct - I wanted to keep this a generic array of objects to make it easier for future changes to the User model.  Plus the Guild model seems to have taken a more generic pattern so I figured it'd be better to assume a similar pattern.